test/unit/definitions/divorce/UserRoleStateAuth.test.js

```diff
@@ -1,27 +1,37 @@
 const expect = require('chai').expect;
 const { differenceWith } = require('lodash');
+const load = require;
+
+function loadAllFiles (files) {
+  let definitions = [];
+
+  files.forEach(file => {
+    definitions = definitions.concat(load(`definitions/divorce/json/AuthorisationCaseState/${file}.json`));
+  });
+
+  return definitions;
+}
 
 const CaseType = Object.assign(require('definitions/divorce/json/CaseType'), []);
 const AuthorisationCaseType = Object.assign(require('definitions/divorce/json/AuthorisationCaseType/AuthorisationCaseType'), []);
 const State = Object.assign(require('definitions/divorce/json/State/State'), []);
-const AuthorisationCaseState = Object.assign(require('definitions/divorce/json/AuthorisationCaseState/AuthorisationCaseState'), []);
 
 const MINIMUM_READ_PERMISSIONS = /C?RU?D?/;
 const EXCLUDED_STATES = ['SOTAgreementPayAndSubmitRequired', 'Rejected', 'Withdrawn', 'DNisRefused', 'solicitorAwaitingPaymentConfirmation', 'AwaitingAmendCase'];
 
-function byCaseType(caseType) {
+function byCaseType (caseType) {
   return entry => {
     return entry.CaseTypeID === caseType;
   };
 }
 
-function byStateName(stateEntry) {
+function byStateName (stateEntry) {
   return stateAuth => {
     return stateAuth.CaseStateID === stateEntry.ID;
   };
 }
 
-function mapErrorArray(caseType) {
+function mapErrorArray (caseType) {
```

Comment:
Did you mean to add the space?

Reply:
auto-formatting :/ 

It's a very old issue that was caused yearssssss ago by Safari (or even Mozzila). For some reason it was required there and it was causing any issues in other browsers (yes, then Javascript was only used in browsers).

TLDR; It was not needed here.